boards/arm/96b_carbon_nrf51/96b_carbon_nrf51.yaml

```diff
@@ -9,3 +9,6 @@ toolchain:
   - gccarmemb
 supported:
   - ble
+testing:
+  ignore_tags:
+    - net
```

Comment:
because networking on the controller will not fit, is not a usecase and does not make sense to build over and over, it is basically the same setup for many nrf51 boards we have. IMO, networking on some of those boards would be enabled on the host side (i.e. the STM on this board), not the controller.